paket.dependencies

```diff
@@ -7,6 +7,8 @@ storage: none
 
 group Build
 
+    framework: netstandard1.6
+    storage: none
```

Comment:
It is to use the global nuget directory. The packages in the root directory is going to be unnecessary.

Reply:
We aren't checking in the packages in the root directory so I think this change makes sense. After all, I think the new *.fsproj file format (and .NET Core) uses the global/cache NuGet directory by default(?)

Reply:
> I think the new *.fsproj file format (and .NET Core) uses the global/cache NuGet directory by default(?)

Yes, the `PackageReference` tag in the  new *.fsproj file is declared to use the global/cache NuGet directory by default.

Reply:
Yes, it's all written in [here](https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders), thank you 😃

>The *global-packages* folder is where NuGet installs any downloaded package. [..] Projects using the PackageReference format always use packages directly from this folder.